x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/malware.tsx

```diff
@@ -189,14 +191,17 @@ export const MalwareProtections = React.memo(() => {
             />
           </h6>
         </EuiTitle>
+        <EuiText color="subdued" size="xs">
```

Comment:
The question longer term would be: should we add this block of JSX manually to every new policy field or have it abstracted behind a component that will determine whether it should be shown or not just by the near existance of that prop name in the "field-to-version" map file. 

Here is what I mean:
We could use a component like this:

```tsx
const SupportedVersionNotice = ({optionName}) => {
  const version = popupVersionsMap.get(optionName);
  if (!version) {
    return null;
  }
  return <EuiText color="subdued" size="xs">
      <FormattedMessage id="...." defaultMessage="Endpoint versions {version}" values={{ version }} />
    </EuiText>
}
```

using it:

```tsx
<SupportedVersionNotice optionName="malware" />
```

And if we ever removed this options from the translation map file, then the text in the UI would just "magically" go way :)

Just an idea.

Reply:
oh that sounds like a noice idea. i can do that